docker-compose.yaml

```diff
@@ -0,0 +1,56 @@
+version: "3.7"
+
+networks:
+  mojaloop-net:
+    name: mojaloop-net
+
+services:
+  reporting-hub-bop-trx-ui:
+    image: mojaloop/reporting-hub-bop-trx-ui:v1.7.2
+    environment:
+      - REACT_APP_API_BASE_URL=http://localhost:3000/
+      - REACT_APP_MOCK_API=false
+    ports:
+      - "8082:8082"
+    networks:
+      - mojaloop-net
+    healthcheck:
+      test: wget -q http://172.17.0.1:8082 -O /dev/null || exit 1
+      timeout: 20s
+      retries: 30
+      interval: 15s
+
+  # reporting-events-processor-svc:
```

Comment:
Can we add some comments here as to why its commented out?

Or should it be commented out?

Same comment applies to mongodb.


I.e. why would someone want to uncomment this stuff out?

Reply:
Done